phpcs.xml.dist

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
+<ruleset name="WP-CLI-maintenance-mode">
+	<description>Custom ruleset for WP-CLI maintenance-mode-command</description>
+
+	<!--
+	#############################################################################
+	COMMAND LINE ARGUMENTS
+	For help understanding this file: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml
+	For help using PHPCS: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Usage
+	#############################################################################
+	-->
+
+	<!-- What to scan -->
```

Comment:
@thrijith Sorry about this - @schlessera correctly remarked on one of my PRs about this and the below comment missing a period. I've updated all my PRs, would you mind adding the periods in this one and your other PR ?